src/lib/pages/home-page-data.ts

```diff
@@ -0,0 +1,41 @@
+import {sanityClient} from 'utils/sanity-client'
+import groq from 'groq'
+
+export async function loadHomePageData() {
+  const slugs = {
+    jumbotron: 'primary-jumbotron-cloudflare-workers-intro',
+  }
+  const data = await sanityClient.fetch(
+    groq`
+      {
+        'jumbotron': ${jumbotronQuery},
+      }
+  `,
+    slugs,
+  )
+
+  return data
+}
+
+const jumbotronQuery = groq`
+*[slug.current == $jumbotron][0]{
+  name,
+  title,
+  description,
+  summary,
+  byline, 
+  meta,
+  path,
+  'slug': resources[][0]->_id,
```

Comment:
This could be written as `'slug': resources[0]->_id,` for more simplicity unless there's a nuance I'm missing.

As far as I can tell, `[]` is a [blank filter that defaults to returning every item in the array](https://www.sanity.io/docs/how-queries-work#some-fine-points-on-arrays-and-projections-649d43d7d179) so in this case we are applying a blank filter and then accessing the first element in the array